ffc-pay-batch-validator/retry.js

```diff
@@ -0,0 +1,16 @@
+const { totalRetries } = require('./config')
+
+async function retry (fn, retriesLeft = totalRetries, interval = 1000, exponential = true) {
```

Comment:
For consistency should this be an expression function instead of a declaration?

Reply:
Agreed.  Updated.